src/features/search/sections/TimeSlot.tsx

```diff
@@ -0,0 +1,35 @@
+import { t } from '@lingui/macro'
+import React from 'react'
+
+import { CenteredSection } from 'features/search/atoms'
+import { _ } from 'libs/i18n'
+import { Range } from 'libs/typesUtils/typeHelpers'
+import { Slider } from 'ui/components/inputs/Slider'
+
+import { useSearch } from '../pages/SearchWrapper'
+
+const MAX_HOUR = 24
+
+const formatHour = (hour: number) => `${hour} h`
```

Comment:
```suggestion
const formatHour = (hour: number) => `${hour}h`
```

il est collé dans les maquettes